src/activity-functions-function-notation.xml

```diff
@@ -531,43 +425,51 @@
                 </statement>
             </webwork>
         </exercise>
-
-        <p>When looking at a graph it's easy to tell if the graph is that of a function or not.  Remember the inputs are found along the horizontal axis while the outputs are found along the vertical axis.  The rule is the same: one input, one output.</p>
-
-        <exercise>
-          <title>Determine if the graph is a function</title>
-          <webwork>
-            <setup>
-              <var name="$nopass">
-                <set>
-                  <member>\text{This graph is a function}</member>
-                  <member correct="yes">\text{This graph is not a function}</member>
-                </set>
-              </var>
-              <var name="$pass">
-                <set>
-                  <member correct="yes">\text{This graph is a function}</member>
-                  <member>\text{This graph is not a function}</member>
-                </set>
-              </var>
-              <pg-code>
-                $nopass = RadioButtons(["This graph is a function","This graph is not a function"],1);
-                $pass = RadioButtons(["This graph is a function","This graph is not a function"],0);
-              </pg-code>
-            </setup>
-            <statement>
-              <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/QEW6GNwr/width/431/height/313/border/888888/sri/true" width="431px" height="313px" style="border:0px;" width="431px" height="313px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
-              <p><var name="$nopass" form="buttons" /></p>
-              <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/NnkVeAwc/width/431/height/313/border/888888/sri/true" width="431px" height="313px" style="border:0px;" width="431px" height="313px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
-              <p><var name="$pass" form="buttons" /></p>
-            </statement>
-          </webwork>
-        </exercise>
-
-        <p>Tables are another way to express inputs and outputs. In a vertical table the inputs are on the left while the outputs are on the right.  In a horizontal table, the top row is the input and the bottom row is the output.</p>
-
+        <p>Tables are one way to express inputs and outputs of a relation. In a vertical table the inputs are on the left while the outputs are on the right.  In a horizontal table, the top row is the input and the bottom row is the output.</p>
+        <p><em>Vertical Table</em></p>
```

Comment:
This looks like tag abuse. This is a title/caption for the table that follows. Try making it a `<title>`, as in `<title>Vertical Table</title>` as the first child of the `<table>`. But I think maybe titles for tables is planned, but not yet implemented. If it doesn't come out, make it a `<caption>` instead. You may not like that the caption appears below instead of above. If you like, later we can move all table captions to the top with a simple CSS change (although this excludes sidebyside tables).

Whatever happens here, do the same for the Horizontal Table that follows.